### PR TITLE
Login and signup page added

### PR DESCRIPTION
**Pull request by SumitSaundadkar**
Login and SignUp page for video library.

### CODE REVIEW

File: src/pages/Login/Login.jsx [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+      <div className="login_container">
+        <form>
+          <h3 className="login_heading">Login</h3>
+          <label>Email</label>
+          <input placeholder="xyz@gmail.com"></input>
+          <div className="login_pass">
+            <label>Password</label>
+            <input type="password" placeholder="•••••••••"></input>
+            <span className="pass_eye">
+              {" "}
```
Comment (TusharJade) [inline src/pages/Login/Login.jsx:16]: You can remove this

**Review by TusharJade — APPROVED** [review]
Everything looks perfect good to go for merge 🚀

File: src/pages/Login/Login.jsx [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+              <i class="fa-solid fa-eye-slash"></i>
+            </span>
+            <div className="form_btn">
+              <button className="btn btn-login ">Login</button>
+              <button className="btn btn-guest">Login as Guest</button>
+              <a href="/signup" className="signupBtn">
```
Comment (aiman-ansari) [inline src/pages/Login/Login.jsx:22]: Instead of anchor tag use Link here

**Review by aiman-ansari — APPROVED** [review]
Code is clean naming could be improve. Remove commented code

**Pull request merged**